zero_bin/rpc/src/lib.rs

```diff
@@ -102,28 +84,33 @@ where
         })
         .take(PREVIOUS_HASHES_COUNT + 1)
         .filter(|i| *i >= 0)
+        .chunks(2)
+        .into_iter()
+        .map(|mut chunk| {
+            // We convert to tuple of (current, Option previous block)
```

Comment:
minor nit, for readability:

```suggestion
            // We convert to tuple of (current_block, Option<previous_block>)
```